app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -188,13 +188,59 @@
+                     :id           => "none_radio",
+                     "ng-checked"  => "emsCommonModel.event_stream_selection == none",
+                     :value        => "none",
+                     "ng-model"    => "emsCommonModel.event_stream_selection",
+                     "ng-change"   => "radioSelectionChanged()",
+                     "checkchange" => ""} None
+            %label.radio-inline.control-label{"for" => "amqp_radio"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "amqp_radio",
+                     "ng-checked"  => "emsCommonModel.event_stream_selection == amqp",
+                     :value        => "amqp",
+                     "ng-model"    => "emsCommonModel.event_stream_selection",
+                     "checkchange" => ""} AMQP
+            %hr
+            %div{"ng-if" => "emsCommonModel.event_stream_selection == 'none'"}
+              Disable event monitoring.
+            %div{"ng-if" => "emsCommonModel.event_stream_selection == 'amqp'"}
+              = render :partial => "layouts/angular-bootstrap/endpoints_angular",
+                       :locals  => {:ng_show                => true,
+                                    :ng_model               => "#{ng_model}",
+                                    :id                     => record.id,
+                                    :prefix                 => "amqp",
+                                    :ng_reqd_api_port       => "false",
+                                    :ng_reqd_hostname       => "true"}
+              = render :partial => "layouts/angular-bootstrap/auth_credentials_angular_bootstrap",
+                       :locals  => {:ng_show           => true,
+                                    :ng_model          => "#{ng_model}",
+                                    :ng_reqd_userid    => true,
+                                    :ng_reqd_password  => true,
+                                    :ng_reqd_verify    => true,
+                                    :validate_url      => validate_url,
+                                    :id                => record.id,
+                                    :prefix            => "amqp",
+                                    :basic_info_needed => true}
         .form-group
-          .col-md-12
+          .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'openstack' || #{ng_model}.emstype == 'openstack_infra'"}
             %span{:style => "color:black"}
               %div{"ng-if" => "emsCommonModel.event_stream_selection == 'amqp'"}
                 = _("Used to authenticate with OpenStack AMQP Messaging Bus for event handling. Configure AMQP if eventing is not enabled on Ceilometer.")
               %div{"ng-if" => "emsCommonModel.event_stream_selection == 'ceilometer'"}
                 = _("Select Ceilometer if eventing is not enabled on AMQP.")
```

Comment:
However, here is the div that might be the one that you are creating above? Can you replicate this in VMware below?